lambda-buffers-compiler/src/LambdaBuffers/Compiler/ProtoCompat/Types.hs

```diff
@@ -311,10 +313,13 @@ data KindCheckError
     RecursiveKindError TyName
   | -- | The following type has the wrong.
     InconsistentTypeError TyName Kind Kind
+  | -- | The following type@(TyName) was redeclared here@(TyName).
+    MultipleTyDefError TyName TyName
```

Comment:
You're missing the `kind` up here.

Reply:
Also don't bother documenting these types. Or if you do than try to be in sync with the Proto docs.